src/components/CompanyHome.jsx

```diff
@@ -14,48 +12,42 @@ class CompanyHome extends Component {
 
     componentDidMount() {
         let company = localStorage.getItem('company')
-        db.collection('companies').doc(company).collection('forms')
+        db.collection('companies').doc(company).collection('Forms')
             .get()
             .then(snapshot => {
                 let companyData = []
```

Comment:
I don't see the need for company data to be an array. You can just set companyData = doc.data() so it's equal to an object. (This way you won't need to say "this.state.companyData[0]" you can just say "this.state.companyData")